### DISCUSSION TITLE
What's the call to free the world ?

### DISCUSSION THREAD

**Discussion by SeleDreams** [SanderMertens/flecs#1376] [Q&A] [answered]
Are we supposed to use a standard free call or does the world have its own free ?

**Comment by copygirl** [comment, marked as answer]
Just for the record, it's `ecs_fini` in C, and in C++ the world destructor calls `release`, which if necessary you can also call manually.

**Reply by SeleDreams** [reply]
Thanks, I actually found out earlier, that's why i closed it.

**Reply by copygirl** [reply]
And other people might see your question, and find it unanswered. Coming across posts online that are unanswered or where the author left a "nevermind, got it" with no other information can be frustrating, so I figured I'd add what I knew.
